nodes/config/ui_base.html

```diff
@@ -1083,7 +1083,7 @@
                 const subflowInstance = n
                 if (subflowDef && subflowDef.env && subflowDef.env.length > 0) {
                     /** @type {{name:String, type:String, value:String}[]} */
-                    const envDefsWithUIGroup = subflowDef.env.filter(env => env.ui.type === 'conf-types' && env.type === 'ui-group')
+                    const envDefsWithUIGroup = subflowDef.env.filter(env => env.type === 'ui-group' && env.ui.type === 'conf-types')
```

Comment:
```suggestion
                    const envDefsWithUIGroup = subflowDef.env.filter(env => env.type === 'ui-group' && env.ui?.type === 'conf-types')
```
Maybe be more defensive here? 

Iirc, it is possible `ui` to not be present in the env.